src/utils/utils.ts

```diff
@@ -177,3 +177,5 @@ export function canAccessFile(file: string) {
     return false;
   }
 }
+
+export const isLocalIpAddress = (ipAdress: string): boolean => ['localhost', '127.0.0.1', '::ffff:127.0.0.1', '::1'].includes(ipAdress);
```

Comment:
Why not a regular function?

Reply:
When do you use arrow-functions and when normal functions? I like to use arrow ones everywhere to have a safe boundary between cases where I need this and not.

Reply:
I usually default to non-arrow functions for top-level utilities as well as methods.

Reply:
Consider adding it to the contributing docs then if regular functions should always be used https://github.com/microsoft/playwright/blob/master/CONTRIBUTING.md#code-style